container/libcontainer/compatibility.go

```diff
@@ -178,8 +178,12 @@ type v1State struct {
 	Config v1Config `json:"config"`
 }
 
-// Relative path to the libcontainer execdriver directory.
-const libcontainerExecDriverPath = "execdriver/native"
+const (
+	// Relative path to the libcontainer execdriver directory.
+	libcontainerExecDriverPath = "execdriver/native"
+	// Absolute path of the containerd directory.
+	containerdPath = "/run/containerd"
```

Comment:
For the future, we should see if we can get all the information we need via containerd APIs.